src/snowflake/connector/config_manager.py

```diff
@@ -189,7 +203,9 @@ def __init__(
         self,
         *,
         name: str,
-        file_path: Path | None = None,
+        file_path: Path
+        | Sequence[tuple[Path, ConfigFileOptions, str | None]]
```

Comment:
This type is pretty complex.  Does it make sense to create a type alias?

Reply:
I did this by declaring it as a NamedTuple